packs/promtail/templates/promtail.nomad.tpl

```diff
@@ -0,0 +1,101 @@
+job [[ template "job_name" . ]] {
+
+  [[ template "region" . ]]
+
+  datacenters = [ [[ range $idx, $dc := .promtail.datacenters ]][[if $idx]],[[end]][[ $dc | quote ]][[ end ]] ]
+  type = "system"
+  
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
+
+  group "promtail" {
+    count = 1
+
+    network {
+      mode = "bridge"
+      port "http" {
+        to = [[ .promtail.http_port ]]
+      }
+    }
```

Comment:
Could we turn this whole section into a variable definition? Along with the constraint variable, it would give people a chance to try and run this on non-linux machine by modifying these. Also some users my have naming conventions for port labels, which would be covered.